xcode/Subconscious/Shared/Library/CombineUtilities.swift

```diff
@@ -41,3 +41,74 @@ struct CombineUtilities {
         .eraseToAnyPublisher()
     }
 }
+
+extension DispatchQueue {
+    /// Run a closure async on this queue, returning a Combine Future.
+    func future<T>(
```

Comment:
Create a Combine future from an async work item on a GDC queue. This helper is similar in principle to the `CombineUtilities.async` helper, but gives us more control over which GDC queue things are run on.

In future, I would like to move all of our uses of `CombineUtilities.async` to `DispatchQueue.future`.